### ISSUE TITLE
Godot 4.3-beta2: "Script export type must be a global class." (modifier_stack.gd, line 15)

### ISSUE THREAD

**Issue by farfalk** [HungryProton/scatter#202]
I'm using the 4.0 release of the plugin within the just released 4.3 beta 2 version of the engine.

The error in title happens everytime I run the project and is also highlighted when I open the mentioned script.

A snippet of `modifier_stack.gd`:
```gdscript
signal stack_changed
signal value_changed
signal transforms_ready


const ProtonScatter := preload("../scatter.gd")
const TransformList := preload("../common/transform_list.gd")
const BaseModifier := preload("../modifiers/base_modifier.gd")

@export var stack: Array[BaseModifier] = []
```

The error is associated to the `@export` line.

(thank you for the awesome addon!)

**Comment by farfalk** [comment]
This is "fixed" by: https://github.com/godotengine/godot/issues/93168#issuecomment-2173127077

Didn't see it when I searched for the error before opening this issue. Sorry.

At least it's now documented on both repos I guess

**Comment by HungryProton** [comment]
Should be fixed in 8e54b326850bb06cae8edd5ccfbbdc8434df39a2
